tools/devtool

```diff
@@ -285,9 +285,17 @@ run_devctr() {
     [[ -t 0 ]] && docker_args+=("-i")
     [[ -t 1 ]] && docker_args+=("-t")
 
+    # Try to pass these environments from host into container for network proxies
+    proxies=(http_proxy HTTP_PROXY https_proxy HTTPS_PROXY no_proxy NO_PROXY)
+    for i in "${proxies[@]}"; do
+        if [[ ! -z ${!i} ]]; then
+            docker_args+=("--env $i=${!i}")
+        fi
+    done
+
     # Finally, run the dev container
     #
-    docker run "${docker_args[@]}" \
```

Comment:
Dropping the double quotes creates a possibly unwanted expansion. For the expected expansion, where each array item expands to one argument, we need to keep the double quotes.